test/ui_test_label_studio.py

```diff
@@ -0,0 +1,213 @@
+        self.driver.switch_to.frame(iframe_)
+        self.driver.find_element_by_xpath(self.Setup_label_car)
+        time.sleep(5)
+        #
+        #Doing 3 clicks for create area
+        action = webdriver.common.action_chains.ActionChains(self.driver)
+        action.move_to_element_with_offset(self.driver.find_element_by_tag_name('body'), 0,0)
+        #
+        # el=self.driver.find_element_by_xpath('/html/body/div/div/div/div/div/div/div[1]/div/div/div[1]/div[2]/div/canvas')
+        # action = webdriver.common.action_chains.ActionChains(self.driver)
+        # action.move_to_element_with_offset(el, 500, 500)
+        # action.click()
+        # action.perform()
+        #
+        x = 700
+        y = 300
+        action.move_by_offset(x, y).click().perform()
+        action.move_by_offset(x, y+50).click().perform()
+        action.move_by_offset(x+30, y+30).click().perform()
+        action.move_by_offset(x, y).click().perform()
```

Comment:
these offsets are relative after each other, so 4th are on 2830 by x axis, for away after screen's boundary :)
(as I know)

Reply:
This is from Setup step, which will be rebuild )